ext_scripts/captcha.php

```diff
@@ -9,7 +9,7 @@ function imageAuthCode($width, $height) {
   $auth_code = md5($auth_code);
   $auth_code = substr($auth_code, 0, 5);
   $auth_code = strtoupper($auth_code);
-  setcookie('image_auth_code', md5($auth_code), time()+60*60 , '/');
+  setcookie('image_auth_code', md5($auth_code), time()+60*60 , '/',"",$_SERVER['HTTPS'] == 'on' ? true : false, true);
```

Comment:
For the HTTPS detection, we should use `isSecure()` by HttpFoundation, see https://github.com/symfony/http-foundation/blob/439fdfdd344943254b1ef6278613e79040548045/Request.php#L1043

It respects also proxies and loadbalancers.
I know, this might be tricky in the captcha here, but maybe we should switch to another captcha lib in this phase.